[2024-07-18 06:35:35,321] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-04-06T00:00:00+00:00 [queued]>
[2024-07-18 06:35:35,394] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-04-06T00:00:00+00:00 [queued]>
[2024-07-18 06:35:35,399] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:35:35,403] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2024-07-18 06:35:35,404] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:35:35,454] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): pull_task> on 2024-04-06T00:00:00+00:00
[2024-07-18 06:35:35,461] {standard_task_runner.py:52} INFO - Started process 3100 to run task
[2024-07-18 06:35:35,482] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_example_dag', 'pull_task', '2024-04-06T00:00:00+00:00', '--job-id', '554', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom.py', '--cfg-path', '/tmp/tmpi6e9u2gn', '--error-file', '/tmp/tmpy8v8wm_n']
[2024-07-18 06:35:35,493] {standard_task_runner.py:77} INFO - Job 554: Subtask pull_task
[2024-07-18 06:35:35,673] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_example_dag.pull_task 2024-04-06T00:00:00+00:00 [running]> on host a868c7bacc12
[2024-07-18 06:35:35,925] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_example_dag
AIRFLOW_CTX_TASK_ID=pull_task
AIRFLOW_CTX_EXECUTION_DATE=2024-04-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-06T00:00:00+00:00
[2024-07-18 06:35:35,946] {logging_mixin.py:104} INFO - Pulled message: Hello from push_function!
[2024-07-18 06:35:35,946] {python.py:118} INFO - Done. Returned value was: None
[2024-07-18 06:35:35,972] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=xcom_example_dag, task_id=pull_task, execution_date=20240406T000000, start_date=20240718T063535, end_date=20240718T063535
[2024-07-18 06:35:36,213] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-18 06:35:36,233] {local_task_job.py:146} INFO - Task exited with return code 0
